src/Marten.NodaTime.Testing/IntegrationContext.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using Marten.Services;
+using Marten.Testing.Harness;
+using Xunit.Abstractions;
+
+namespace Marten.NodaTime.Testing
+{
+    public abstract class IntegrationContext : IDisposable
```

Comment:
**CONCERN:** Cannot we reuse the `IntegrationContext` class from the main project? Maybe we could create a separate base project with base classes to reuse them between different test projects?

Reply:
It wasn't playing nicely here, and I was lazy. I don't see it doing any harm